project_resource_calendar/models/instrument.py

```diff
@@ -10,21 +10,16 @@ class Instrument(models.Model):
     _inherit = ['resource.resource']
     _description = __doc__
 
-    currency_id = fields.Many2one(
-        'res.currency',
-        string='Currency',
-    )
-    tarification = fields.Monetary(
+    tarification = fields.Float(
```

Comment:
we should put tarification in 'resource.resource' because it's also used in room